src/reducers/index.js

```diff
@@ -131,6 +131,7 @@ function reducers(state = getInitialState(), action) {
       if (!shallowEqual(state, newState) || scrollAnchor.changed) {
         newState = computeRenderedRows(newState, scrollAnchor);
       }
+      newState.scrollJumped = state.scrollJumped || scrollAnchor.changed;
```

Comment:
This may be true on any scroll change.  Should we name it something like scrollChanged instead of jumped?